omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,42 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend>Provider Account</legend>
+        <table>
+            <tr>
+                <td>
+                    <c:if test="${isProvider == true}">
+                        <table cellspacing="15">
+                            <thead>
+                                <tr>
+                                    <th style="text-align:center"> Name </th>
+                                    <th style="text-align:center"> Identifier </th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <c:forEach items="${providerList}" var="currentProvider">    
+                                    <tr>
+                                       <td align="center"> <c:out value="${currentProvider.getName()}"/> </td>
+                                       <td align="center"> <c:out value="${currentProvider.getIdentifier()}"/> </td>
+                                    </tr>
+                                </c:forEach>
+                            </tbody>
+                        </table>
+                    </c:if>
+                    
+                    <c:if test="${isProvider == false}">
+                        <input name="providerCheckBox" type="checkbox" value="addToProviderTable"/>
+                        <label for="providerCheckBox">Create a Provider account for this User</label>
```

Comment:
Have a look at my comment [here](https://github.com/openmrs/openmrs-module-legacyui/pull/41/files#discussion_r54492052) and on the ticket discussion between me and Darius.